packages/models-library/tests/test_basic_types.py

```diff
@@ -0,0 +1,18 @@
+import pytest
+from faker import Faker
+from models_library.basic_types import UUIDStr
+from pydantic import ValidationError
+from pydantic.tools import parse_obj_as
+
+
+@pytest.mark.skip(reason="DEV: testing parse_obj_as")
+def test_parse_uuid_as_a_string(faker: Faker):
```

Comment:
what is the use of that test?

Reply:
I was verifying how parse_obj_as works (as the mark says) ...should i move it elsewhere?